content/applications/finance/accounting/payables/supplier_bills/invoice_digitization.rst

```diff
@@ -0,0 +1,78 @@
+=============================================================
+Invoice digitization with optical character recognition (OCR)
+=============================================================
+
+**Invoice digitization** is the process of automatically encoding traditional paper invoices into
+vendor bills and customer invoices forms in your accounting.
+
+Odoo uses OCR and artificial intelligence technologies to recognize the content of the documents.
```

Comment:
If you want to give a try to the abbreviation markup, here is a good place!
the text remains OCR, but it is underlined with dots, and you can see the full meaning as a tooltip, when you hover on top of "OCR" with the mouse.

```suggestion
Odoo uses :abbr:`OCR (optical character recognition)` and artificial intelligence technologies to recognize the content of the documents.
```